plugins/inputs/mock/mock.go

```diff
@@ -0,0 +1,160 @@
+package mock
+
+import (
+	"math"
+	"math/rand"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Mock struct {
+	counter int64
+
+	MetricName string            `json:"metric_name"`
+	Tags       map[string]string `json:"tags"`
+
+	Random   []*Random   `json:"random_float"`
+	Step     []*Step     `json:"step"`
+	Stock    []*Stock    `json:"stock"`
+	SineWave []*SineWave `json:"sine_wave"`
```

Comment:
Can you please detail on why you are using JSON tags instead of the usual `toml` annotations?

Reply:
Habit :) I'll clean these up